conf/modules.config

```diff
@@ -119,28 +119,36 @@ process {
             pattern: "*vcf.gz.tbi"
         ]
     }
+}
 
 // MAPPING
-    withName: 'BWAMEM2_MEM_T' {
+process {
+    withName: "BWAMEM1_MEM_T|BWAMEM2_MEM_T" {
```

Comment:
Does this generate any weird warnings when the mapper is not used?

Reply:
I haven't check that out yet

Reply:
So I checked, and no, no weird warnings